### PR TITLE
Feature/homepage review

### PR DESCRIPTION
**Pull request by nicklee**
Updates to bring the homepage inline with the fixed left navigation.

Based on https://github.com/wagtail/guide/pull/120 (120 needs to be merged in first)

### CODE REVIEW

File: apps/frontend/static_src/scss/components/introduction.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.introduction {
+    font-size: 20px;
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/components/introduction.scss:2]: rem – for later #129

File: apps/frontend/static_src/scss/components/headings.scss [hunk 2]
```diff
@@ -15,4 +15,10 @@
             background-color: $color--black;
         }
     }
+
+    &--display {
+        font-size: 90px;
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/components/headings.scss:20]: rem – for later #129

File: apps/frontend/static_src/scss/components/section.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                transition: transform 0.1s;
+            }
+        }
+    }
+
+    &__icon {
+        padding-right: $gutter;
+        flex-basis: 16.665%;
+        min-width: 16.666%;
+        max-width: 16.666%;
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/components/section.scss:27]: I’m surprised those are different values. Might be worth a variable and a bit of math? Is that like 60% of 10?

File: apps/frontend/static_src/scss/components/section.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    &__heading {
+        color: $color--black;
+        font-size: 30px;
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/components/section.scss:32]: rem – for later #129

File: apps/frontend/static_src/scss/components/section.scss [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        margin-bottom: ($gutter * 0.5);
+        font-weight: 700;
+        transition: color $transition;
+    }
+
+    &__description {
+        color: $color--grey;
+        font-size: 16px;
```
Comment (thibaudcolas) [inline apps/frontend/static_src/scss/components/section.scss:40]: rem – for later #129

**Review by thibaudcolas — APPROVED** [review]
👍

File: apps/frontend/static_src/scss/components/section.scss (hunk 3 above)
Comment (nicklee) [reply apps/frontend/static_src/scss/components/section.scss]: oops, a typo :) thanks

**Pull request merged**